rs/config/src/embedders.rs

```diff
@@ -124,7 +150,7 @@ impl FeatureFlags {
             write_barrier: FlagStatus::Disabled,
             wasm_native_stable_memory: FlagStatus::Enabled,
             wasm64: FlagStatus::Enabled,
-            best_effort_responses: FlagStatus::Disabled,
+            best_effort_responses: BestEffortResponsesFeature::FallBackToGuaranteedResponse,
```

Comment:
Shouldn't we still keep it at `DisabledByTrap` here and then do the switch in behavior with an explicit PR?

Reply:
Yeah, agreed. I guess one advantage of having it never trap is that we can also right away merge the tests which start sending best effort messages.